ui/lib/core/addon/components/kv-suggestion-input.hbs

```diff
@@ -0,0 +1,31 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+<div ...attributes {{did-update this.updateSuggestions @mountPath}}>
+  <FormFieldLabel @label={{@label}} @subText={{@subText}} />
+  <FilterInput
```

Comment:
Are these inputs overlayed? As in the user inputs to FilterInput and power select handles the dropdown options? A comment may be helpful since looking at the code it looks like two inputs render one after the other

Reply:
Yes I am passing the `is-hidden` class onto the trigger component since the `FilterInput` is being used to trigger the dropdown instead. PowerSelect has the ability to define a different component to use as the trigger which would involve using the component helper and passing it as the `triggerComponent` arg. The problem is that the component helper does not support passing attributes to a glimmer component which the `FilterInput` makes heavy use of. This could be worked around by creating a wrapper component but that sets the attributes internally and the ones that are dynamic could then be passed as component args. That just didn't feel great to me so I opted to hide the trigger instead.

I'll go ahead and add a comment so that it's clear that the `FilterInput` is being used as the trigger.